README.md

```diff
@@ -1,22 +1,114 @@
-# Little Esty Shop
+# LittleEstyShop
 
-## Background and Description
+Welcome to LittleEstyShop! This project is a web application that serves as a platform for merchants to manage their items and invoices. It also provides admins with tools to monitor merchant performance and handle invoices. The application consumes the Unsplash API to display images, including the app logo, item images, merchant images, and random photos on the merchant's dashboard.
```

Comment:
Aren't the merchant images & random photos on merchant's dashboard the same thing?